http-servers/src/main/java/com/palantir/remoting/http/server/ExceptionMappers.java

```diff
@@ -29,10 +29,9 @@ private ExceptionMappers() {}
 
     public static void visitExceptionMappers(boolean includeStackTrace,
             Consumer<ExceptionMapper<? extends Throwable>> consumer) {
-        consumer.accept(new ForbiddenExceptionMapper(includeStackTrace));
         consumer.accept(new IllegalArgumentExceptionMapper(includeStackTrace));
         consumer.accept(new NoContentExceptionMapper());
-        consumer.accept(new NotFoundExceptionMapper(includeStackTrace));
         consumer.accept(new RuntimeExceptionMapper(includeStackTrace));
+        consumer.accept(new WebApplicationExceptionMapper(includeStackTrace));
```

Comment:
I was nervous about the ordering here (i.e. I didn't want ``RuntimeException``s to be handled first), but this seems to work. I guess something clever must happen with finding the most specific matching ``ExceptionMapper``.